vendor/github.com/mindprince/gonvml/bindings_nocgo.go

```diff
@@ -0,0 +1,115 @@
+// +build !cgo
```

Comment:
Seems like we might need a `// +build cgo` in the actual cgo bindings?
looks like the bazel build generation is including both? :disappointed:

Reply:
The `// #cgo LDFLAGS: -ldl` directive imports the file when built with CGO enabled. https://golang.org/cmd/cgo/#hdr-Using_cgo_with_the_go_command

Reply:
yeah, I don't think gazelle (generating the build files) is aware of this.

Reply:
nevermind, gazelle handles this fine. sorry for the noise https://github.com/kubernetes/kubernetes/pull/82342#discussion_r320950134